lib/Models/Terria.ts

```diff
@@ -151,6 +151,10 @@ interface ConfigParameters {
    * The URL to use for Cesium terrain in the 3D Terrain viewer, in quantized mesh format. This property is ignored if "useCesiumIonTerrain" is set to true.
    */
   cesiumTerrainUrl?: string;
+  /**
+   * The Cesium Ion Asset ID to use for Cesium terrain in the 3D Terrain viewer. `cesiumIonAccessToken` will be used to authenticate. This property is ignored if "useCesiumIonTerrain" is set to true.
+   */
+  cesiumIonAssetId?: number;
```

Comment:
Should we name this as `cesiumTerrainAssetId` like you have added to doc/customizing/client-side-config.md